omniscapeImpact-2.0.0/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package
+  name="omniscapeImpact"
+  version="2.0.0"
+  description="Calculates changes in connectivity categories from a baseline"
+  url="https://github.com/ApexRMS/omniscapeImpact"
+  location="https://github.com/ApexRMS/omniscapeImpact/releases/download/2.0.0/omniscapeImpact-2-0-0.ssimpkg">
+
+  <onlineLibrary name="omniscapeImpact-example" displayName="OmniscapeImpact Example"
```

Comment:
Just a couple points here:
* Fix indenting to match other package metadata
* Use omniscapeImpact package display name in template displayName and description